docs/Concepts/Privacy/Plugin-Privacy.md

```diff
@@ -0,0 +1,64 @@
+---
+description: Privacy Plugin
+---
+
+# Privacy Plugin
+The Privacy Plugin allows users to define their own strategy for private transactions. 
+This could take many forms, but ultimately it's down to the plugin implementor to provide besu with a private transaction when required.
+
+!!! important
+
+    Plugin privacy is an early access feature.
+
+    Plugin interfaces are subject to change between releases. 
+
+## Configuration
+Plugin Privacy must be enabled using `--Xprivacy-plugin-enabled` and a plugin must be registered that implements a `PrivacyPluginPayloadProvider`.
```

Comment:
Hi @antonydenyer, is there a URL or additional info we can link to for `PrivacyPluginPayloadProvider`?

I'm unable to find anything in the [Plugin API docs](https://javadoc.io/doc/org.hyperledger.besu/plugin-api/latest/index.html)

Reply:
Not yet - waiting on https://github.com/hyperledger/besu/pull/2509 to be merged. 

https://github.com/hyperledger/besu/blob/5edb3b105ca31228fb8d4429d8866eb8298dc9f3/plugin-api/src/main/java/org/hyperledger/besu/plugin/services/privacy/PrivacyPluginPayloadProvider.java

Reply:
https://github.com/hyperledger/besu/issues/2102